team.html

```diff
@@ -122,56 +140,63 @@ <h4 class="info">Jing Li</h4>
                 <div class="row group">
                     <h3>MAYA</h3>
                     <div class="col-md-2 profile">
-                        <img src="images/team/Thomas_Storwick.jpg"/>
-                        <h4 class="info">Thomas Storwick</h4>
+                        <img src="images/team/Isaac_Harris.png"/>
+                        <h4 class="info">Isaac Harris</h4>
                         <h6>Team Lead</h6>
                     </div>
+                   
                     <div class="col-md-2 profile">
-                        <img src="images/team/Isaac_Harris.png"/>
-                        <h4 class="info">Isaac Harris</h4>
+                        <img src="images/team/Tom_Jensen-Large.jpg">
+                    	<h4 class="info">Tom Jensen-Large</h4>
                     </div>
                     <div class="col-md-2 profile">
-                    	<img src="images/team/Ciara_Azam.jpg">
-                        <h4 class="info">Ciara Azam</h4>
+                        <img src="images/team/Jane Wang.png">
```

Comment:
Some of these aren't updated. Can we get an updated roster before we change this?